tensorboard/uploader/upload_tracker.py

```diff
@@ -341,9 +342,12 @@ def send_tracker(self):
             yield
         finally:
             self._update_cumulative_status()
+            if self._one_shot:
+                scan_complete_msg = "Done scanning through logdir"
```

Comment:
Can you add a new test in this file in the vicinity of this line?

https://github.com/tensorflow/tensorboard/blob/master/tensorboard/uploader/upload_tracker_test.py#L253

Reply:
Done.

Reply:
I don't see a change to update_tracker_test.py. Did you forget to commit and push the change?